### ISSUE TITLE
Single point line not visible

### ISSUE THREAD

**Issue by DennisNurtio** [imaNNeo/fl_chart#1438]
**Describe the bug**
A line chart with only one point will not show anything. I would expect at least a dot that shows a data point is added to the chart. 

Scenario: our customer is setting up sensors. After setup the chart is shown (with a hourly interval) so the first point shows after/while installing. The next point would be visible after an hour. Currently the customer does not see any data in the first hour, and might wonder why the setup failed.

**To Reproduce**
The following code shows as an empty chart (while there is a data point available).
```
@override
  Widget build(BuildContext context) {
    return LineChart(
      LineChartData(
        lineBarsData: [
          LineChartBarData(
            color: AppColors.contentColorGreen,
            barWidth: 8,
            isStrokeCapRound: true,
            dotData: const FlDotData(show: false),
            belowBarData: BarAreaData(show: false),
            spots: const [
              FlSpot(1, 1),
            ],
          ),
        ],
        minX: 0,
        maxX: 14,
        maxY: 4,
        minY: 0,
      ),
      duration: const Duration(milliseconds: 250),
    );
  }
```

**Screenshots**
![image](https://github.com/imaNNeo/fl_chart/assets/70628966/3b85ac6b-3965-40b4-9d84-d724e6ff803a)

**Versions**
 - which version of the Flutter are you using? 3.13.1
 - which version of the FlChart are you using? 0.63.0

**Comment by FMorschel** [comment]
In this case, you could do `dotData: const FlDotData(show: true)` I believe. Tested here and it works just fine. Would this be a problem for you? If so, why?

**Comment by DennisNurtio** [comment]
Hello @FMorschel , thanks for the tip. Currently we dont show the dots, because it does not match the styling. For this specific case I will show it, as a good workaround.
`dotData: const FlDotData(show: spots.length == 1)`

**Comment by imaNNeo** [comment]
In the new versions, it shows. Can you please verify that?

**Comment by imaNNeo** [comment]
@DennisNurtio Can you please provide a reproducible code?

**Comment by DennisNurtio** [comment]
Pasted the original code above into line 12 of `lib\presentation\samples\line\line_chart_sample1.dart` on commit `e3910de`. 
Same effect as the original post.

**Comment by imaNNeo** [comment]
I found the bug, thank you!

**Comment by imaNNeo** [comment]
Landed in [0.65.0](https://github.com/imaNNeo/fl_chart/releases/tag/0.65.0)
Check it out!
